tamr_unify_client/client.py

```diff
@@ -144,8 +144,9 @@ def set_auth_cookie(self):
             "./instance:login",
             json={"username": self.auth.username, "password": self.auth.password},
         )
-        if r.status_code != 200:
-            raise Exception  # TODO: Replace with sensible handling
+        # Only continue if request is successful
```

Comment:
nit: I think this comment doesn't say anything not said by the code. I'd recommend removing it or replacing with something that explains _why_ we are doing this conceptually if you think its worth explaining.